lib/graphql/analysis/ast/analyzer.rb

```diff
@@ -25,7 +25,9 @@ def result
           raise NotImplementedError
         end
 
-        # Don't use make_visit_method because it breaks `super`
+        private
```

Comment:
This won't actually make it private since the method is defined as `self.build_visitor_hooks`. You have two options:

```ruby
def self.build_visitor_hooks(member_name)
  # etc
end

private_class_method :build_visitor_hooks

# or even weirder

private_class_method def self.build_visitor_hooks(member_name)
  # etc
end
```

```ruby
class << self
  private

  def build_visitor_hooks(member_name)
    # etc
  end
end
```

Reply:
Whoops, thanks for this! 🔥Updated!